app/templates/orders/order_block.hbs

```diff
@@ -1,31 +1,82 @@
-<li>
+<li class="order_block">
 
-    <div class="row state_details">
-      <div class="small-5 columns">
-        {{order.code}} - {{order.status}}
-      </div>
-      <div class="small-4 columns">
-        {{order.designatedOrdersPackages.length}} <i class="fa fa-shopping-basket"></i>
-        {{order.dispatchedOrdersPackages.length}} <i class="fa fa-ship"></i>
+    <div class="row order_header">
+
+      <div class="small-4 columns order_transport"> 
+        <i class="fa fa-lg fa-{{ order.transportIcon }}"></i> {{ order.transportLabel }}
       </div>
-      <div class="small-3 columns">
-        {{display-datetime order.createdAt format="DD MMM 'YY"}}
+
+      <div class="small-3 columns order_code">
+        {{order.code}}
       </div>
-    </div>
 
-    <div class="row">
-      <div class="small-12 columns">
-        {{order.organisation.name}}
+      <div class="small-5 columns order_state">
+        <div class="order_state_icon {{ order.state }}">
+          <i class="fa fa-{{ order.stateIcon }}"></i>
+        </div>
+        <div class="order_state_text"> {{ t (concat "order." order.state) }} </div>
       </div>
+
     </div>
 
-    <div class="row">
-      <div class="small-12 columns">
-        {{order.stockitContact.fullName}}: {{order.stockitContact.displayNumber}}
-        {{#if order.localOrder.clientName}}
-          for {{order.localOrder.clientName}}
-        {{/if}}
-      </div>
+    <div class="row order_body">
+
+      {{#if order.orderTransport.scheduledAt }}
+        <div class="row">
+          <div class="small-4 columns field">
+            {{ t "order.due_date" }}
+          </div>
+          <div class="small-8 columns value">
+            {{  display-datetime order.orderTransport.scheduledAt format="dddd Do MMMM hh:mm a" }}
+          </div>
+        </div>
+      {{/if}}
+
+      {{#if order.submittedBy }}
+        <div class="row">
+          <div class="small-4 columns field">
+            {{ t "order.submitted_by" }}
+          </div>
+          <div class="small-8 columns value">
+            {{ order.submittedBy.firstName }} {{ order.submittedBy.lastName }}
+          </div>
+        </div>
+      {{/if}}
+      
+      {{#if order.gcOrganisationId or order.organisationId }}
+        <div class="row">
+          <div class="small-4 columns field">
+            {{ t "organisation_title" }}
+          </div>
+          <div class="small-8 columns value">
+            {{ if order.isGoodCityOrder order.gcOrganisation.nameEn order.organisation.name }}
```

Comment:
@patrixr , I'm having trouble understanding this condition. Can you please explain?

Reply:
@Darshan-Chauhan this condition extracts the name from the gc_organisation only if it is a good city order, otherwise uses the stockit_organisation name